app/controllers/audio_files_controller.rb

```diff
@@ -31,10 +40,27 @@ def destroy
   @audio_file = AudioFile.find(params[:id])
   @audio_file.destroy
 
+  if @audio_file.nil?
```

Comment:
This conditional needs to move up just before line 41 so we can guard against `@audio_file` being nil there.

Reply:
Also, to test deleting an invalid audio file:

(Shortcut in Chrome to highlight elements in the dev tools: `Ctrl+Shift+C`)

![how_to_test_deleting_invalid_audio_files](https://user-images.githubusercontent.com/185602/95283868-960f8d80-0811-11eb-922e-dd1a315a1ece.gif)